int32 move_type
string from_sqr
string to_sqr
char moved_piece
char captured_piece
---
bool success
string message